hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,550 @@
+---
+title: Tutorial--Make a simple photo viewer that targets multiple platforms
+description: Learn how to reach users on other platforms like Web, iOS, and Android with minimal changes to the C#/WinUI 3 simple photo viewer built in the previous tutorial. We'll use Uno Platform to create a new multi-platform app, which we can move code from the existing desktop project to.
+ms.topic: article
+ms.date: 05/21/2023
+keywords: Windows, App, SDK, WinUI 3, WinUI, photo, viewer, Windows 11, Windows 10, XAML, C#, uno platform, uno
```

Comment:
should other platforms be added?

Reply:
@sasakrsmanovic This is something I'll check with Alvin about